bin/cloudgov/audit/db-integrity

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
+
+#
+# Requires sqlvalidator, run pip install sqlvalidator if this doesn't work.
+#
+
+# we might be running in circleci
+if [ -f /home/circleci/project/env.local ]; then
+  . /home/circleci/project/env.local
+fi
+# we might be running from a local dev machine
+SCRIPT_DIR="$(dirname "$0")"
+if [ -f $SCRIPT_DIR/env.local ]; then
+  . $SCRIPT_DIR/env.local
+fi
+if [ -f ./env.local ]; then
+  . ./env.local
+fi
+if [ -f $SCRIPT_DIR/../../deploy/includes ]; then
+  . $SCRIPT_DIR/../../deploy/includes
+else
+   echo Cannot find $SCRIPT_DIR/../../deploy/includes
+   exit 1
+fi
+
+# just testing?
+if [ x$1 = x"--dryrun" ]; then
```

Comment:
`--dryrun` doesn't work. I think it would be fine to just delete this clause (that is, don't support `--dryrun`). Having it here is misleading.